versions/2016-10-31.md

```diff
@@ -1273,3 +1274,34 @@ CorsConfiguration:
       ExposeHeaders:
         - "*"
 ```
+
+### Referable properties of SAM resources
+- [AWS::Serverless::Function](#referable-properties-of-serverless-function)
+- [AWS::Serverless::Api](#referable-properties-of-serverless-RestApi)
+- [AWS::Serverless::HttpApi](#referable-properties-of-serverless-HttpApi)
+
+#### Referable properties of Serverless Function
+Property Name | Reference | LogicalId | Description
+---|:---:|---|---
+Alias | `function-logical-id`.Alias | `function-logical-id`Alias`alias-name` | SAM generates an `AWS::Lambda::Alias` resource when `AutoPublishAlias` property is set. This resource can be referenced in intrinsic functions by using the resource logical ID or  `function-logical-id`.Alias
+Version | `function-logical-id`.Version | `function-logical-id`Version`sha` | SAM generates an `AWS::Lambda::Version` resource when `AutoPublishAlias` property is set. This resource can be referenced in intrinsic functions by using the resource logical ID or  `function-logical-id`.Version
+Destination (for SNS Topic) | `function-logical-id`.DestinationTopic |`function-logical-id`EventInvokeConfig`OnSuccess/OnFailure`Topic| SAM auto creates an `AWS::SNS::Topic` resource when `Destination` property of `DestinationConfig` property in `EventInvokeConfig` property is not specified. This generated resource can be referenced by using `function-logical-id`.DestinationTopic
```

Comment:
I would make the property name `DestinationTopic`

Reply:
Thank you, I will update